pkg/apis/coordination/install/install.go

```diff
@@ -34,5 +35,6 @@ func init() {
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(coordination.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion))
+	utilruntime.Must(v1.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(v1.SchemeGroupVersion, v1beta1.SchemeGroupVersion))
```

Comment:
prioritize v1beta1 over v1 for storage until v1 has been available for at least one release:

`scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1.SchemeGroupVersion)`

Reply:
done